plugins/modules/one_vnet.py

```diff
@@ -213,8 +213,8 @@
     type: int
     returned: when O(state=present)
     sample: 1
-vm_mad:
-    description: The network's VM_MAD.
+vn_mad:
```

Comment:
Please note that changing the name of the return value is a breaking change and must come with a very long deprecation period.

Reply:
Alright, surely I understand the importance of the reverse compatibility but for that particular case I think we can ignore it, as previous value was part of a bug :)

Reply:
The problem is that users out there may be using that return value and their playbooks will start breaking without any notice. My suggestion: keep the old variable name `vm_mad` in the output, deprecate it, and add `vn_mad` to the output. After deprecation period, we remove the old one.

Reply:
> The problem is that users out there may be using that return value and their playbooks will start breaking without any notice. My suggestion: keep the old variable name `vm_mad` in the output, deprecate it, and add `vn_mad` to the output. After deprecation period, we remove the old one.

Hello! I doubt it as this module just doesn't work because of this parameter :) and also It was added in 9.4.0 so wouldn't it be better just to change right away?

Reply:
@felixfontein Hello! Could you please guide me what shoud I do? Is it really that important to do that deprecation process as without these changes module simply doesn't work, please also note that It was added in 9.4.0 so most probably no one got to use it anyway (I guess if anyone have used it then there would probably be an issue created :) )

Reply:
> this module just doesn't work because of this parameter

Do you mean that it is impossible to get the module to finish due to this return value? Can you provide more information on that?

If the module still works, it's basically what @russoz said: add the correct return value, deprecate the existing (wrong) one.

Reply:
yes, as I mentioned in the initial topic there was also a typo in `'vm_mad': template.VM_MAD,` which led to an error in all cases so perhaps I could change name of return value too?